oauthlib/oauth1/rfc5849/signature.py

```diff
@@ -307,12 +286,12 @@ def collect_parameters(uri_query='', body=[], headers=None,
     #
     #     *  The entity-body follows the encoding requirements of the
     #        "application/x-www-form-urlencoded" content-type as defined by
-    #        `W3C.REC-html40-19980424`_.
+    #        W3C.REC-html40-19980424 `W3C-HTML-4.0`_.
 
     #     *  The HTTP request entity-header includes the "Content-Type"
     #        header field set to "application/x-www-form-urlencoded".
     #
-    # .._`W3C.REC-html40-19980424`: https://tools.ietf.org/html/rfc5849#ref-W3C.REC-html40-19980424
+    # .. _`W3C-HTML-4.0`: https://www.w3.org/TR/1998/REC-html40-19980424/
```

Comment:
why did you change this?

Reply:
The shorter reference identifier is more easily recognisable as "HTML 4.0". The "-40-19980424" makes it harder for the human reader and adds little value in this context. Also, to satisfy PEP-8 maximum line length rule.